internal/hash/handler.go

```diff
@@ -0,0 +1,24 @@
+package hash
+
+import (
+	"context"
+	"github.com/mach-composer/mach-composer-cli/internal/config"
+	"github.com/mach-composer/mach-composer-cli/internal/graph"
+	"os"
+)
+
+const defaultHashFile = ".mach/hashes.json"
```

Comment:
Should we use `.mach-composer/hashes.json` ?

Reply:
I think we used `.mach` in the removed lockfile thing, but I will change it to mach-composer instead

Reply:
ah it got used by the github checkout we do. So the usage of this folder is a bit spread out